Signal/src/view controllers/CallViewController.swift

```diff
@@ -244,8 +244,12 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
         declineIncomingButton = createButton(imageName:"hangup-active-wide",
                                              action:#selector(didPressDeclineCall))
 
+        let spacerView = UIView(frame: CGRect(x: 0, y: 0, width: buttonSize(), height: 0))
+        spacerView.autoSetDimension(.width, toSize:buttonSize())
+        spacerView.autoSetDimension(.height, toSize:buttonSize())
+
         incomingCallView = createContainerForCallControls(controlGroups : [
-            [acceptIncomingButton, declineIncomingButton ]
+            [acceptIncomingButton, spacerView, declineIncomingButton ]
```

Comment:
rowWithSubviews() Already creates spacers!  

It's busted because we currently only make sure the spacers all have the same width.  I suggest we replace:
```
spacer.autoMatch(.width, to:.width, of:lastSpacer!)
```
with something like:
```
// Only use spacing for two-button rows.
let spacerWidth = subviews.count == 2 ? ScaleFromIPhone5To7Plus(10, 20) : 0
spacer.autoSetDimension(.width, spacerWidth)
```